src/components/datepicker/DatepickerTable.vue

```diff
@@ -53,6 +57,38 @@
                     index++
                 }
                 return visibleDayNames
+            },
+
+            hasEvents() {
+                return this.events && this.events.length
+            },
+
+            /*
+            * Return array of all events in the specified month
+            */
+            eventsInThisMonth() {
+                if (!this.events) return []
+
+                const monthEvents = []
+
+                for (let i = 0; i < this.events.length; i++) {
+                    let event = this.events[i]
+
+                    if (!event.hasOwnProperty('date')) {
+                        event = { date: event }
+                    }
+                    if (!event.hasOwnProperty('type')) {
+                        event.type = 'is-primary'
+                    }
+                    if (Object.prototype.toString.call(event.date) !== '[object Date]') {
+                        event.date = new Date(event.date)
+                    }
+                    if (event.date.getMonth() === this.focused.month && event.date.getUTCFullYear() === this.focused.year) {
```

Comment:
Datepicker component use `getFullYear()` to check a date (as you can see in `DatepickerTableRow.vue`)